src/component/axis/CartesianAxisView.ts

```diff
@@ -135,6 +135,13 @@ const axisElementBuilders: Record<typeof selfBuilderAttrs[number], AxisElementBu
             tickModel: splitLineModel
         });
 
+        if (splitLineModel.get('showMinLine') === false) {
```

Comment:
We usually use `!splitLineModel.get('showMinLine')` to check false value.

Reply:
d